### ISSUE TITLE
Feature Request: Anchor destinations for edges connecting nodes.

### ISSUE THREAD

**Issue by kdog3682** [Jollywatt/typst-fletcher#21]
It would be really nice to have a way to specify where (with regard to a node) an arrow should point at.

**Sample Code:**

```
#diagram({
   node((0,0), [hi])
   edge(from: "north", to: "south-west") // coming from the north of the previous node and pointing to the southwest of the destination node.
   node((3, 3), [hi]) 

})
```

**Comment by Jollywatt** [comment]
Does something like the following do what you mean?
```typ
#import "@preview/fletcher:0.4.3": diagram, node, edge

#diagram({
  node((0,0), [start])
  edge("n,e,sssss,e,ne") // line goes [n]orth from previous node, then east, south 5 units, east, then north-east into the next node
  node((3,3), [end])
})
```
You have to describe the exact path yourself, though.

**Comment by Jollywatt** [comment]
Basic cetz anchors are now implemented in v0.5.3 (ca3ecde057551ef396acd24c76c1cec7511ef714), allowing:
```typ
#diagram({
   node((0,0), [hi], name: <a>)
   edge(<a.north>, <b.south-west>)
   node((3, 3), [hi], name: <b>)
})
```
